stablehlo/tests/CheckOps.cpp

```diff
@@ -156,6 +157,46 @@ llvm::Error evalExpectSerializedEqOp(const Tensor &expected, StringRef probeId,
   return evalExpectEqOp(expected, *tensor);
 }
 
+inline std::string toString(const Element &elem) {
```

Comment:
IMO we do not need this given we have 
https://github.com/openxla/stablehlo/blob/d5a22dd393e685ffeeed952e906136f0f860e74c/stablehlo/reference/Element.cpp#L1126.

wdyt?

Reply:
`print` gives a more verbose output. For instance, includes element type that in this particular case is known (e.g. `[1.57079637 : f32, -89.4159851 : f32]`). Also, `print` outputs non-finite numbers using hex format (e.g. `0xFF800000` for `Inf`).

Considering the use case here, I am fine eliminating `toString`.

Reply:
Now using print.